File: app/src/main/java/com/cherry/testopencv/MainActivity.java
package com.cherry.testopencv;

import androidx.appcompat.app.AppCompatActivity;

import android.graphics.Bitmap;
import android.graphics.BitmapFactory;
import android.os.Bundle;
import android.widget.ImageView;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        Bitmap bitmap = BitmapFactory.decodeResource(getResources(), R.drawable.sam);
        Bitmap bitmap1 = ImageProcess.processFace(bitmap);
        ImageView imageView = findViewById(R.id.image_view);
        if (bitmap1 != null)
            imageView.setImageBitmap(bitmap);
    }

}

File: app/src/main/java/com/cherry/testopencv/ui/MainActivity.kt
package com.cherry.testopencv.ui

import android.os.Bundle
import android.view.View
import androidx.appcompat.app.AppCompatActivity
import com.cherry.testopencv.R

open class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        window.decorView.systemUiVisibility = View.SYSTEM_UI_FLAG_FULLSCREEN

//        val binding = DataBindingUtil.setContentView<ActivityMainBinding>(this, R.layout.activity_main)
//        // use this ViewModelProvider will cache the ViewModel data , Even though Activity has been destory.
//        // MainViewModel.Factory() this factory you can input some args you want .
//        val viewModel: MainViewModel = ViewModelProvider(this, MainViewModel.Factory()).get(MainViewModel::class.java)
////        val mainViewModel = MainViewModel()
//        binding.viewModel = viewModel
    }

}